middle_end/flambda2/terms/flambda_primitive.ml

```diff
@@ -1512,11 +1549,12 @@ type ternary_primitive =
   | Bytes_or_bigstring_set of bytes_like_value * string_accessor_width
   | Bigarray_set of num_dimensions * Bigarray_kind.t * Bigarray_layout.t
   | Atomic_compare_and_set
+  | Array_vector_set of Lambda.boxed_vector * Array_set_kind.t
```

Comment:
Same remark as above, do we really need the `Array_set_kind.t` here, or can can we always recompute it from the `boxed_vector` info ?